packages/nx-ignore/src/index.ts

```diff
@@ -36,14 +36,23 @@ if (!project) {
   process.exit(1);
 }
 
-console.log(
-  `≫ Using Nx to determine if this project (${project}) is affected by the commit...`
-);
-
 main();
 
 async function main() {
+  const commitMessage = execSync(`git log -1 --pretty='%B'`).toString();
+
+  if (commitHasSkipMessage(commitMessage)) {
+    console.log(`🛑 - Skip build due to commit message: ${commitMessage}`);
+    process.exit(0); // this tells Vercel to skip build
```

Comment:
I know its a bit silly, but since we have process.exit + the comment explaining it in a few places now, can we move it to a function like this?

```
function exitAndBuild() {}
function exitAndSkipBuild() {}
```

This would also provide a good spot to be able to modify the script to work on other vendors (e.g. netlify) if they happen to use a different status code or method for marking the builds.

Reply:
Sure, I mean we can always change it later too. But I get your point. :)